src/beamlime/handlers/detector_data_handler.py

```diff
@@ -232,7 +233,7 @@ def _cleanup(self) -> None:
 _registry = {
     'geometry-dream-2025-01-01.nxs': 'md5:91aceb884943c76c0c21400ee74ad9b6',
     'geometry-loki-2025-01-01.nxs': 'md5:8d0e103276934a20ba26bb525e53924a',
-    'geometry-bifrost-2025-02-14.nxs': 'md5:b581f1560aefd4bec56c86395f553516',
+    'geometry-bifrost-2025-01-01.nxs': 'md5:ae3caa99dd56de9495b9321eea4e4fef',
```

Comment:
Couldn't get the previous file to work, something about missing `depends_on`. I think just replacing it is fine?